editoast/openapi.yaml

```diff
@@ -199,7 +199,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/InternalError'
-  /health:
+  /health/:
```

Comment:
Is that intended? Most routes have the same change, which doesn't make sense to me

Reply:
I believe that's a side-effect of the macro changes.

It doesn't really matter as we trim any trailing slash in a request anyway.

We should probably do it to when generating the openapi.yml

Reply:
Done.